modules/git/notes_gogit.go

```diff
@@ -36,6 +41,7 @@ func GetNote(ctx context.Context, repo *Repository, commitID string, note *Note)
 			remainingCommitID = remainingCommitID[2:]
 		}
 		if err != nil {
+			log.Error("Unable to find git note corresponding to the commit %q. Error: %v", commitID, err)
```

Comment:
if err == object.ErrDirectoryNotFound  it's not an error we should log

Reply:
Aah yes. I should move this statement and place it after the if block.